### PR TITLE
ability to change default queue (server side swift)

### PR DESCRIPTION
**Pull request by k1313**

### CODE REVIEW

File: Sources/DispatchQueue+Promise.swift [hunk 1]
```diff
@@ -38,16 +38,6 @@ extension DispatchQueue {
...
      */
-    class public final var `default`: DispatchQueue {
-        get {
-            return __PMKDefaultDispatchQueue()
-        }
-        set {
-            __PMKSetDefaultDispatchQueue(newValue)
-        }
-    }
+    static public var `default`: DispatchQueue = DispatchQueue.main
```
Comment (mxcl) [inline Sources/DispatchQueue+Promise.swift]: This means the objc value for this function will be different to the Swift version.

Comment (k1313) [inline Sources/DispatchQueue+Promise.swift]: Are these values the same in the current version?

Comment (mxcl) [inline Sources/DispatchQueue+Promise.swift]: I don't understand this question, sorry.

Comment (mxcl) [inline Sources/DispatchQueue+Promise.swift]: Yes they are the same, the code you removed is what makes them the same…

Comment (k1313) [inline Sources/DispatchQueue+Promise.swift]: sorry, don't know obj-c :)

Comment (k1313) [inline Sources/DispatchQueue+Promise.swift]: it's ok now?

**Comment by mxcl** [comment]
PromiseKit 5 allows this.

**Pull request closed without merging**